### PR TITLE
push-and-use-china-registry

### PR DESCRIPTION
**Pull request by calvix**
towards giantswarm/giantswarm#7435

push and use china image in china region

### CODE REVIEW

**Comment by calvix** [comment]
work fine, my only question is should i bump the version ?

**Comment by kopiczko** [comment]
> work fine, my only question is should i bump the version ?

Do you need to release it?

File: helm/aws-operator/values.yaml [hunk 1]
```diff
@@ -7,9 +7,12 @@ flag:
       labelselector:
         version: ""
 image:
-  registry: "quay.io"
   name: "giantswarm/aws-operator"
   tag: "[[ .Version ]]"
+Installation:
+  V1:
+    Registry:
+      Domain: quay.io
```
Comment (kopiczko) [inline helm/aws-operator/values.yaml]: Is that needed? The default is set here https://github.com/giantswarm/installations/blob/91d748669d9523f53d285f3cafde1812d7165261/default-draughtsman-configmap-values.yaml#L718

**Comment by kopiczko** [comment]
BTW here's how it's done in net-exporter https://github.com/giantswarm/net-exporter/pull/77/files

File: helm/aws-operator/values.yaml (hunk 1 above)
Comment (calvix) [reply helm/aws-operator/values.yaml]: Yes, for e2e test this is needed as the e2e installations tools dont use draightsman and dont have its defaults

Comment (kopiczko) [reply helm/aws-operator/values.yaml]: Understood. Then we should unify the approach with https://github.com/giantswarm/net-exporter/pull/77/files

**Comment by calvix** [comment]
probably don't need specific release as any real change will pick this up and thats fine

**Comment by kopiczko** [comment]
Yes, that's also what I thought.

File: helm/aws-operator/values.yaml (hunk 1 above)
Comment (rossf7) [reply helm/aws-operator/values.yaml]: The problem is cert-exporter and net-exporter are deployed by both draughtsman and app CRs. So `image.registry` is needed for app CRs.

aws-operator is deployed by app CR but for VOO so only the draughtsman values are used. So I think `.Values.Installation.V1.Registry.Domain` is correct.

I find this confusing and maybe supporting both structures is better and more future proof?

Comment (calvix) [reply helm/aws-operator/values.yaml]: Yes, what @rossf7  said. I did that already to few places. I woudl leave it that way.

Comment (kopiczko) [reply helm/aws-operator/values.yaml]: 👍

**Pull request merged**